sympy/concrete/expr_with_intlimits.py

```diff
@@ -13,6 +13,14 @@ def __init__(self, expr, msg):
             "%s could not be reordered: %s." % (expr, msg))
 
 class ExprWithIntLimits(ExprWithLimits):
+    """
+    The class for expressions with integer limits.
```

Comment:
This should specify that it is the superclass of Sum and Product.

Reply:
See https://github.com/sympy/sympy/pull/17698/files#diff-50952a7e18ce9c5bd3f6b4b0299b8174

Reply:
I have made an update. Please take a look.